blueprints/factories/dataplex-datascan-factory/README.md

```diff
@@ -0,0 +1,159 @@
+# Dataplex DataScan Factory
+
+<!-- BEGIN TOC -->
+  - [Rule Templates](#rule-templates)
+- [Variables](#variables)
+<!-- END TOC -->
+
+## Overview
+
+This module allows creation and management of Dataplex DataScan resources based on a directory of well-formatted YAML configuration files files. YAML abstraction enable data SMEs to create data profile and data quality scans without needing to understand Terraform.
+
+This factory is based on the [Dataplex DataScan module](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/tree/master/modules/dataplex-datascan). The module can be forked and customized to your organization's needs.
+
+## Example
+
+Given the following directory:
+
+```bash
+datascan-specs
+├── table-1
+│   ├── data_profile.yaml
+├── table-2
+│   ├── data_quality.yaml
+```
+
+The yaml files are as follows. Documentation for the variables accepted in a DataScan config YAML file is provided in `sample-specs/example.yaml`.
+
+```yaml
+# tftest-file id=profile path=datascan-specs/table-1/data_profile.yaml
+data:
+  resource: "//bigquery.googleapis.com/projects/bigquery-public-data/datasets/austin_bikeshare/tables/bikeshare_stations"
+data_profile_spec:
+```
+
+```yaml
+# tftest-file id=dq path=datascan-specs/table-2/data_quality.yaml
+data:
+  resource: "//bigquery.googleapis.com/projects/bigquery-public-data/datasets/austin_bikeshare/tables/bikeshare_stations"
+data_quality_spec:
+  sampling_percent: 100
+  row_filter: "station_id > 1000"
+  rules:
+    - column: address
+      dimension: VALIDITY
+      ignore_null: null
+      non_null_expectation: {}
+```
+
+With this file structure, we can use the factory as follows:
+
+```hcl
+module "datascan-factory" {
```

Comment:
Optional: Would be nice to have another factory usage example at the bottom which uses the defaults + template behavior as well. 

That way users could see an out-of-the-box example of all the features in the sample-specs folder.